tools/serve/serve.py

```diff
@@ -979,6 +1015,8 @@ def build_config(logger, override_path=None, config_cls=ConfigBuilder, **kwargs)
             raise ValueError("%s path %s does not exist" % (title, value))
         setattr(rv, key, value)
 
+    rv.polyfill_url = kwargs.get("polyfill_url");
```

Comment:
```suggestion
    rv.polyfill_url = kwargs.get("polyfill_url")
```

This seems to be the reason the checks are failing